Solutions/SentinelOne/Data Connectors/SentinelOne_API_FunctionApp.json

```diff
@@ -3,7 +3,7 @@
 	"title": "SentinelOne",
 	"publisher": "SentinelOne",
 	"descriptionMarkdown": "The [SentinelOne](https://www.sentinelone.com/) data connector provides the capability to ingest common SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API. Refer to API documentation: `https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview` for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more.",
-	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. Follow the steps to use this Kusto functions alias **SentinelOne** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-SentinelOneAPI-parser).",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected.",
```

Comment:
These queries are dependent on a parser based on a Kusto Function deployed as part of the solution. Please update this text.